components/table/index.en-US.md

```diff
@@ -135,6 +135,29 @@ One of the Table `columns` prop for describing the table's columns, Column has t
 | -------- | ----------- | ---- | ------- |
 | title | Title of the column group | string\|ReactNode | - |
 
+### pagination
+
+Properties for pagination.
+
+| Property | Description | Type | Default |
+| -------- | ----------- | ---- | ------- |
+| current | current page number | number | - |
+| defaultCurrent | default initial page number | number | 1 |
+| defaultPageSize | default number of data items per page | number | 10 |
+| hideOnSinglePage | Whether to hide pager on single page | boolean | false |
+| itemRender | to customize item innerHTML | (page, type: 'page' \| 'prev' \| 'next', originalElement) => React.ReactNode | - |
+| pageSize | number of data items per page | number | - |
+| pageSizeOptions | specify the sizeChanger options | string\[] | ['10', '20', '30', '40'] |
+| position | specify the position of `Pagination` | 'top' \| 'bottom' \| 'both' | 'bottom' |
```

Comment:
Don't need copy all Pagination props to here. Just leave `positon` here, and link others to `/components/pagination-cn/#API`.